### ISSUE TITLE
The denoiser plugin does not support headless mode and will not be enabled 2.4+0.4.0-pre.1

### ISSUE THREAD

**Issue by jackjt8** [chunky-dev/chunky-denoiser#20]
```
Loading plugin: "chunky-denoiser.jar"
The plugin DenoiserPlugin was developed for Chunky 2.4.0-159-g8ed94bb7 but this is Chunky 2.4.0-183-gffad4f42 - it may not work properly.
The denoiser plugin does not support headless mode and will not be enabled.
Plugin loaded: DenoiserPlugin 0.4.0-pre.1
```

Headless denoising still not working. Was meant to be resolved with 0.4.0-pre.1.

**Comment by MadDog443** [comment]
I just did a pull request which should disable the check.

**Comment by MadDog443** [comment]
Yep, headless still doesn't work right.

**Comment by leMaik** [comment]
Fixed by #28
